src/stream/stream.service.js

```diff
@@ -1,3 +1,5 @@
+/* global moment */
```

Comment:
since the module depends on 'angular-moment', you can avoid this by injecting 'moment' where it's needed